Addresses issue 175 (#221)
DE-RSE/2023_paper-RSE-groups: paper.tex

Also, having this expertise centralised allows for synergies in the deployment procedure for mobile apps:
The RSE unit can create institutional accounts with the app stores and manage the time-consuming deployment process including hard-to-setup procedures like code signing.
Besides the technical benefits of this central deployment procedure, the institution benefits from the increased visibility and the ability to build a brand with its technological output.

> Similar advantages to having this expertise centralised are relevant to other institutions where research software development happens without any interaction between units, regardless of the type of applications developed or the scientific domain.

Reviewer: ```suggestion
Similar advantages to having this expertise centralised are relevant to other institutions where silos could occur, regardless of the type of applications developed or the scientific domain.
```
